devservices/config.yml

```diff
@@ -24,6 +24,12 @@ x-programs:
 services:
   gateway:
     image: ghcr.io/getsentry/conduit-gateway:latest
+    environment:
+      GATEWAY_PORT: ${GATEWAY_PORT:-8000}
+      REDIS_URL: ${REDIS_URL:-redis://redis:6379}
+      JWT_ISSUER: ${JWT_ISSUER:-}
```

Comment:
FYI If you don't have a default, you can just leave the value empty and it should take the env variables from shell or .env file. This should work too tho